src/components/Engines/Engines.test.tsx

```diff
@@ -1,71 +1,66 @@
 import React from 'react';
 import { mount } from 'enzyme';
 
+import { DetailContext } from '../../pages/Version';
+import { PackageMetaInterface } from '../../../types/packageMeta';
+
 import Engine from './Engines';
 
 jest.mock('./img/node.png', () => '');
 jest.mock('../Install/img/npm.svg', () => '');
 
-const mockPackageMeta: jest.Mock = jest.fn(() => ({
+const mockPackageMeta = (): PackageMetaInterface => ({
```

Comment:
IMO this should be an object, not a func.

Reply:
The code was already there, in this case, we can ignore this part for now.

Reply:
Using a function ensures the object is new in each test, preventing accidental contamination caused by mutations.

Reply:
I've changed this now - the function takes a param to set up the `engine` prop in the test data